contracts/message-based/PriceAware.sol

```diff
@@ -26,6 +26,10 @@ contract PriceAware is Ownable {
     emit TrustedSignerChanged(trustedSigner);
   }
 
+  function isSignerAuthorized(address _receviedSigner) internal virtual view returns (bool) {
```

Comment:
If we're adding this method we may remove the getTrustedSigner one to reduce the complexity and gas costs:
_receviedSigner == trustedSigner

Reply:
I would leave it to be compatible with our previous approaches. If we override the `isSignerAuthorized` function the `getTrustedSigner` should not even be called so it shouldn't consume a lot of gas for just existing.